pkg/kubectl/autoscale.go

```diff
@@ -38,6 +39,7 @@ func (HorizontalPodAutoscalerV1Beta1) ParamNames() []GeneratorParam {
 		{"scaleRef-kind", false},
 		{"scaleRef-name", false},
 		{"scaleRef-apiVersion", false},
+		{"scaleRef-apiGroup", false},
```

Comment:
I would just find it *very* confusing if the `apiVersion` field here is a groupless version, completely unlike the `apiVersion` at the top level of an API object